training

```diff
@@ -273,215 +273,250 @@ for _round in $(seq -f "%02g" 0 99); do
 
-            cat sub_new$round.$TOPIC \
-                | sort \
-                | uniq \
-                | join - rel.$TOPIC.fil -v1 \
-                | cut -d' ' -f1 \
-                | sed -e 's/^/-1 /' > x_negat.$round
+                cat sub_new$round.$TOPIC \
+                    | sort \
+                    | uniq \
+                    | join - rel.$TOPIC.fil -v1 \
+                    | cut -d' ' -f1 \
+                    | sed -e 's/^/-1 /' > x_negat.$round
+
+                cat  x_negat.* \
+                    | sort -k2 \
+                    | join - ../$JUDGECLASS.svm.fil.svd -2 1 -1 2 > seed_ssarp_docs.$TOPIC
 
-            cat  x_negat.* \
-                | sort -k2 \
-                | join - ../$JUDGECLASS.svm.fil.svd -2 1 -1 2 > seed_ssarp_docs.$TOPIC
+                cat  x_posit.* \
+                    | shuf -n 1 \
+                    | sort -k2 \
+                    | join - ../$JUDGECLASS.svm.fil.svd -2 1 -1 2 >> seed_ssarp_docs.$TOPIC
 
-            cat  x_posit.* \
-                | shuf -n 1 \
-                | sort -k2 \
-                | join - ../$JUDGECLASS.svm.fil.svd -2 1 -1 2 >> seed_ssarp_docs.$TOPIC
+                cut -d ' ' -f2- seed_ssarp_docs.$TOPIC  > seed_ssarp_labels.$TOPIC
 
-            cut -d ' ' -f2- seed_ssarp_docs.$TOPIC  > seed_ssarp_labels.$TOPIC
+                cp x_posit.$round sub_new_positives.$round
+                cp x_negat.$round sub_new_negatives.$round
 
-            cp x_posit.$round sub_new_positives.$round
-            cp x_negat.$round sub_new_negatives.$round
+                ssarp_relevants=`wc -l < x_posit.$round`
 
-            ssarp_relevants=`wc -l < x_posit.$round`
+            ) 2> $STD_ERROR_OUT
 
+            catch || {
+                kill -SIGUSR1 `ps --pid $$ -oppid=`; exit_on_error
+            }
         #-----------------------------------------------------------------
         #
         #-----------------------------------------------------------------
         else
 
-            $VERBOSE && e_secondary "\tCreating files..."
-
-            # TODO: Check if the files temp_posit and temp_negat
-            # could be united into a single file.
-
-            cat sub_new$round.$TOPIC \
-                | sort \
-                | uniq \
-                | join - rel.$TOPIC.fil \
-                | cut -d' ' -f1 \
-                | sed -e 's/^/1 /' > temp_posit.$round.$TOPIC
-
-            cat sub_new$round.$TOPIC \
-                | sort \
-                | uniq \
-                | join - rel.$TOPIC.fil -v1 \
-                | cut -d' ' -f1 \
-                | sed -e 's/^/-1 /' > temp_negat.$round.$TOPIC
+            try
+            (
+                e_secondary "\tCreating files..."
+
+                # TODO: Check if the files temp_posit and temp_negat
+                # could be united into a single file.
+
+                cat sub_new$round.$TOPIC \
+                    | sort \
```

Comment:
I've added the following dummy and erroneous code block to the beginning of `try` blocks:
```sh
python3 meuadawibdwun
cp file-not-exists9645964594654
commandNOTfound123
```

But the code executed normally (no errors detected). I used this block in near all `try` blocks, but none worked, this is probably a error in `try()` function: It expects 2 or 3 arguments, but none were given. A suggestion is to mark them as optional for now, and later I will correct this behaviour (since I introduced this bug :disappointed:)